cmd/admin-heal.go

```diff
@@ -715,6 +738,16 @@ func mainAdminHeal(ctx *cli.Context) error {
 		return nil
 	}
 
+	if prefix == "" && opts.Recursive && opts.Pool == nil && opts.Set == nil && isTerminal() {
```

Comment:
```suggestion
	if prefix == "" && opts.Recursive && opts.Pool == nil && opts.Set == nil && isTerminal() {
```

You can also do this for all prefixes as well @vadmeste only doesn't report this for `pool and set` are set since a prefix may be as big as the entire namespace. 

Also we have can `--force` to override this `y/n`